Control/Monad/Logic.hs

```diff
@@ -73,7 +73,7 @@ newtype LogicT m a =
     LogicT { unLogicT :: forall r. (a -> m r -> m r) -> m r -> m r }
 
 -------------------------------------------------------------------------
--- | Extracts the first result from a LogicT computation,
+-- | Extracts the first result from a @LogicT@ computation,
```

Comment:
While we are at it, could you please use `'` instead of `@`? This will hyperlink identifiers to their definitions.